README.md

```diff
@@ -112,7 +112,13 @@ Its work is hosted and supported by the [Node.js Foundation](https://nodejs.org/
 
 ## React
 
-[React](https://facebook.github.io/react/) section to be completed.
+[React](https://facebook.github.io/react/) is a library developed and used at Facebook for building user interfaces. It can be seen as the V in MVC as it makes no assumptions about the rest of the technology stack. Using React Native it can even be used to power native apps.
```

Comment:
Add *(Model View Controller)* after *MVC*.
Maybe a link to React Native? Would be interesting. To an entry of ours if we have one, or to a relevant resource.